packages/botonic-nlp/tests/tasks/text-classification/process/prediction-processor.test.ts

```diff
@@ -0,0 +1,18 @@
+import { tensor } from '@tensorflow/tfjs-node'
+
+import {
+  Intent,
+  PredictionProcessor,
+} from '../../../../src/tasks/text-classification/process/prediction-processor'
+import { OutputData } from '../../../../src/tasks/text-classification/process/types'
+import * as helper from '../../../helpers/tasks/text-classification/constants-helper'
```

Comment:
Try always to import only the necessary code you need:
`import {CLASSES} from '....../constants-helper'`